ROMFS/px4fmu_common/mixers/vtol_AERT.aux.mix

```diff
@@ -0,0 +1,30 @@
+Mixer for a horizontal format with X-Quad and tractor AERT plane configuration
```

Comment:
Well, AERT would mean the pin numbering as in how to plug them into Pixhawk (unless I'm terribly wrong), so:
1: aileron
2: elevator
3: rudder
4: throttle